devimint/src/devfed.rs

```diff
@@ -376,12 +375,6 @@ impl DevJitFed {
             "too many offline nodes ({offline_nodes}) to reach consensus"
         );
 
-        std::env::set_var(FM_GWID_CLN_ENV, self.gw_cln().await?.gateway_id().await?);
-        std::env::set_var(FM_GWID_LND_ENV, self.gw_lnd().await?.gateway_id().await?);
-        if let Some(gw_ldk) = self.gw_ldk().await? {
-            std::env::set_var(FM_GWID_LDK_ENV, gw_ldk.gateway_id().await?);
-        }
```

Comment:
For reviewers: these are not used anywhere meaningful and they trigger a polling loop by contacting each gateway. Just not necessary.